packages/core/src/typegenTypes.ts

```diff
@@ -104,40 +101,6 @@ export interface ResolvedTypegenMeta extends TypegenMeta {
 
 export type TypegenConstraint = TypegenEnabled | TypegenDisabled;
 
-// if combined union of all missing implementation types is never then everything has been provided
-export type AreAllImplementationsAssumedToBeProvided<
```

Comment:
older versions of integration packages still refer to this so it might be best to just `@deprecate` it

Reply:
Older integration packages don't work with XState v5 anyway

Reply:
> unless you mean minor/patch versions from the current major?

This